src/burn/drv/neogeo/neo_run.cpp

```diff
@@ -4513,8 +4519,14 @@ INT32 NeoRender()
 
 inline static void NeoClearOpposites(UINT8* nJoystickInputs)
 {
+	// 0x03 to Binary = 00000011 (Up and Down enabled)
+	// 0x0C to Binary = 00001100 (Left and Right enabled)
+	// SOCD Option in Inputs config would be appreciated.
+	// Standard SOCD Config is Up + Down = Up, Left+Right = Neutral
+	bool SOCD = false;

```

Comment:
you can also use `static constexpr bool SOCD = false` to help the compiler recognize the `if` branch below as "always execute"